pkgs/applications/editors/emacs/default.nix

```diff
@@ -33,6 +33,7 @@ stdenv.mkDerivation rec {
   name = "emacs-${version}${versionModifier}";
   version = "26.3";
   versionModifier = "";
+  appName = "Emacs";            # used by Darwin systems in emacs/wrapper.nix
```

Comment:
Wouldn't it be better to accept this as an argument to the function that returns the derivation and stick the attribute in `passthru`?
That way changing the `appName` would not cause a rebuild.